tools/docker/docker-compose-dev.yaml

```diff
@@ -70,12 +70,11 @@ services:
         condition: service_healthy
       postgres:
         condition: service_healthy
+    volumes:
+      - '../../:/app/src:z'
 
   eda-default-worker:
-    image: 'localhost/eda-default-worker'
-    build:
-      context: ../../
-      dockerfile: tools/docker/worker/Dockerfile
```

Comment:
Since we are removing the Dockerfile, we might need to add this command here:
```
command:
    command:
      - aap-eda-manage
      - rqworker
      - --worker-class
      - aap_eda.core.tasking.DefaultWorker
```

Reply:
added